applications/nrf5340_audio/src/bluetooth/le_audio_cis_headset.c

```diff
@@ -501,6 +501,8 @@ static void disconnected_cb(struct bt_conn *conn, uint8_t reason)
 
 	LOG_INF("Disconnected: %s (reason 0x%02x)", addr, reason);
 
+	ble_mcs_discover_called_set(false);
```

Comment:
IMO: THis needs to be renamed as this will always be set to false.

Reply:
Removed this function